esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -644,10 +582,7 @@ int CWsESDLConfigEx::publishESDLBinding(const char * bindingName,
         return -1;
     }
 
-    if (!ensureSDSPath(ESDL_BINDINGS_ROOT_PATH))
-        throw MakeStringException(-1, "Unexpected error while attempting to access ESDL definition dali registry.");
-
-    Owned<IRemoteConnection> conn = querySDS().connect(ESDL_BINDINGS_ROOT_PATH, myProcessSession(), RTM_LOCK_WRITE, SDS_LOCK_TIMEOUT_DESDL);
+    Owned<IRemoteConnection> conn = querySDS().connect(ESDL_BINDINGS_ROOT_PATH, myProcessSession(), RTM_LOCK_WRITE | RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT_DESDL);
```

Comment:
So in this case, it comes down to how much overhead we're taking on due to the RTM_CREATE_QUERY, because there's always the chance that the /ESDL/Bindings tree in dali was removed by some other process... 

Publishing a new Binding shouldn't happen often, and we want to ensure the request completes even if the dali branch was deleted, so it seems worthwhile to keep the CREATE_QUERY.. thoughts?